elements/pfe-cta/demo/pfe-cta.html

```diff
@@ -180,7 +180,7 @@ <h4 style="color:white;">Color override</h4>
     </div>
   </div>
 
-  <div class="custom-dark-band resize">
+  <div color-palette="darkest" class="custom-dark-band resize">
```

Comment:
I think `color-palette` attribute was supposed to be removed in the revert commit.

```Suggestion
<div class="custom-dark-band resize">
```

Reply:
I agree with removing the `--context`.  These approaches sound resonable to me.  In the future, we could always provide a dedicated declarative component for opting in to the ContextProvider / ContextConsumer.